src/index.ts

```diff
@@ -47,7 +48,7 @@ export const handleTokenRequest = async (ctx: Context, request: Request) => {
 	}
 
 	const keyID = tokenRequest.truncatedTokenKeyId.toString();
-	const key = await ctx.cache.ISSUANCE_KEYS.get(keyID);
+	const key = await asyncRetries(ctx, ctx.cache.ISSUANCE_KEYS.get)(keyID);
```

Comment:
To safely pass methods of a class, one has to bind the class instance before passing the method to the wrapper function
https://javascript.info/bind#solution-2-bind


```suggestion
	const key = await asyncRetries(ctx, ctx.cache.ISSUANCE_KEYS.get.bind(ctx.cache.ISSUANCE_KEYS))(keyID);
```

Reply:
this was the reason to use arrow function attributes
can do both